R/display.R

```diff
@@ -4,17 +4,54 @@
 #'
 #' @return div which wraps your message to display it in the top-right corner of your shiny app.
 #' @export
-display <- function(message) {
-  style = "
-    position: absolute;
-    top: 0;
-    right: 0;
-    width: auto;
-    height: auto;
-    background-color: #E0E0E0;
-    padding: 3px 8px;
-    font-size: 12px;
-    z-index = 9999;"
+#' @import glue
+display <- function(message, position = "top right") {
+  allowed_positions <- c("top right", "top left", "bottom right", "bottom left")
+  if (!position %in% allowed_positions)
+    error("Position argument not allowed.")
+  splitted_position <- unlist(strsplit(position, " "))
+  position_vertical <- splitted_position[1]
+  position_horizontal <- splitted_position[2]
+  tagList(
+    tags$head(
+      tags$style(
+        HTML(paste0(
+        "
+        #shinyinfo {
```

Comment:
This CSS code is not very convenient to read because of combining `paste0` and `glue`. I think it will be more readable if you:

1. Use `glue` + double braces https://github.com/tidyverse/glue#a-literal-brace-is-inserted-by-using-doubled-braces
2. Add indentation in css classes definition

Reply:
Thanks for that tip! I wasn't aware of this double `{` glue syntax.